windows/src/developer/TIKE/oskbuilder/UframeTouchLayoutBuilder.pas

```diff
@@ -208,133 +208,23 @@ procedure TframeTouchLayoutBuilder.FormDestroy(Sender: TObject);
 end;
 
 procedure TframeTouchLayoutBuilder.ImportFromKVK(const KVKFileName: string);   // I3945
```

Comment:
This content in this function and the next one, `SetupModifierKeysForImportedLayout`, has been refactored into a new unit, `Keyman.Developer.System.TouchLayoutToVisualKeyboardConverter`, which is used by TIKE and kmconvert.